server/app.js

```diff
@@ -6,15 +6,18 @@ const { session } = require('./middleware/session');
 const embedApp = require('./apps/embed');
 const authRoutes = require('./routes/auth');
 const testRoutes = require('./routes/tests');
+const automationSchedulerRoutes = require('./routes/automation');
 const path = require('path');
 const apolloServer = require('./graphql-server');
+const setupMockAutomationSchedulerServer = require('./tests/util/mock-automation-scheduler-server');
 const app = express();
 
 // test session
 app.use(session);
 app.use(bodyParser.json());
 app.use('/auth', authRoutes);
 app.use('/test', testRoutes);
+app.use('/jobs', automationSchedulerRoutes);
```

Comment:
Since `jobs` is a fairly open-ended term, could we call this `automation-jobs`?

Reply:
This might need to be addressed in the follow-up that removes or alters the REST API. I will flag your comment in that task.